client/state/plugins/recommended/reducer.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { combineReducers, keyedReducer } from 'state/utils';
+import { RECOMMENDED_PLUGINS_REQUEST, RECOMMENDED_PLUGINS_RECEIVE } from 'state/action-types';
+
+export const items = keyedReducer( 'siteId', ( state = [], action ) => {
```

Comment:
If we default this reducer to `null`, we could totally use that case to get rid of the `isRequesting` reducer and selectors completely. This could also help us remove the `.items` leaf of the reducer tree, as we'll be left with just this single reducer.

Reply:
Great idea!